src/components/organisms/ItemForm/ItemFormContainer.jsx

```diff
@@ -23,12 +23,14 @@ export default compose(
       const regex = /[|&;$%@"<>()+#,' ']/
       if (regex.test(value)) {
         setFileNameError('* Please avoid using special characters or spaces.')
+      } else if (!value) {
```

Comment:
@tbrandle I think we should avoid checking for empty string with just !value here, would be best to do `value.trim() === ''` or a similar comparison to make sure it isn't completely empty, or just whitespace